### PR TITLE
Added fmtlib.

### PR DESCRIPTION
**Pull request by Rinnegatamante**

### CODE REVIEW

File: fmt/VITABUILD [hunk 1]
```diff
@@ -0,0 +1,20 @@
+pkgname=fmt
+pkgver=6.2.0
+pkgrel=1
+url="https://opus-codec.org/"
```
Comment (isage) [inline fmt/VITABUILD:4]: that's wrong

File: fmt/VITABUILD [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+source=("https://github.com/fmtlib/fmt/releases/download/$pkgver/$pkgname-$pkgver.zip")
+sha256sums=('48e03526ba87ef9cf5f1c47b5ebe3aa195bd89b912a57060c36184a6cd19412f')
+depends=('libogg' 'opus')
```
Comment (isage) [inline fmt/VITABUILD:7]: and that one too

**Comment by d3m3vilurr** [comment]
is vitasdk only compatible with 6.2.0? how about 6.2.1 & 7.x?

**Comment by Rinnegatamante** [comment]
I used 6.2.0 since it's the version used in EasyRPG toolchain (so tested working).

**Pull request merged**